cpp/autosar/test/rules/A7-1-2/test.cpp

```diff
@@ -138,9 +138,9 @@ class DerivedClass : public virtual VirtualBaseClass {
   DerivedClass(int i) = delete;           // COMPLIANT
   DerivedClass(int i, LiteralClass lc) {} // COMPLIANT
 private:
-  static int m1; // NON_COMPLAINT - static member variable can be constexpr
+  int m1 = 0;
```

Comment:
Needs "NON_COMPLIANT" marker

Reply:
Fixed in [e9b3ebd](https://github.com/github/codeql-coding-standards/pull/794/commits/e9b3ebd1c9e4e92d83ac47a6a65f119709e0a5a5)